Cargo.toml

```diff
@@ -88,6 +89,8 @@ avif = ["avif-encoder"]
 avif-encoder = ["ravif", "rgb"]
 # Non-default, even in `avif`. Requires stable Rust and native dependency libdav1d.
 avif-decoder = ["mp4parse", "dcv-color-primitives", "dav1d"]
+# Enable serde for dependent crates.
+serde = ["dep:serde", "rgb/serde", "num-complex/serde", "num-rational/serde"]
```

Comment:
Use `rgb?/serde` to avoid pulling in the optional `rgb` dependency unintentionally.

Coincidentally, `rgb?/serde` is the official example for [optional dependency features](https://doc.rust-lang.org/cargo/reference/features.html#dependency-features).

(I am not a maintainer)